metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-tea",
-  "version": "0.2.0",
+  "version": "0.2.4",
```

Comment:
please don't adjust the version number. we will take care of it during the release process.